src/tests/users.test.js

```diff
@@ -38,13 +41,143 @@ describe('User tests', () => {
 
   it('it should not register the user if they dont provide a unique email', async () => {
     const res = await testBase.post('/users/register').send({
-      userName: 'user1',
+      userName: 'user10',
       email: 'user1@gmail.com',
       password: '12345678',
       firstName: 'user',
       lastName: 'one',
     });
     expect(res.status).toEqual(400);
-    expect(res.body).toEqual({ message: 'User already registered' });
+    expect(res.body).toEqual({ message: 'Email already in use' });
+  });
+
+  it('it should not register the user if they dont provide a unique username', async () => {
+    const res = await testBase.post('/users/register').send({
+      userName: 'user1',
+      email: 'user11@gmail.com',
+      password: '12345678',
+      firstName: 'user',
+      lastName: 'one',
+    });
+    expect(res.status).toEqual(400);
+    expect(res.body).toEqual({ message: 'Username already in use' });
+  });
+
+  it('should get the user profile', async () => {
+    const res = await testBase.get('/users/profile/user1');
+    expect(res.status).toEqual(200);
+    expect(res.body).toMatchObject({
+      _id: uid1,
+      userName: 'user1',
+      email: 'user1@gmail.com',
+    });
```

Comment:
would be nice if you added this stuff to fixtures/mocks to prevent sudden typos in some situations